platform-service/src/unstract/platform_service/controller/platform.py

```
@@ -70,28 +69,30 @@ def get_organization_from_bearer_token(token: str) -> tuple[int | None, str]:
+        return False
 
-        platform_key_table = DBTable.PLATFORM_KEY
-        query = f"""
-            SELECT * FROM \"{Env.DB_SCHEMA}\".{platform_key_table}
-            WHERE key = '{token}'
-        """
+    platform_key_table = DBTable.PLATFORM_KEY
+    query = f"""
+        SELECT * FROM \"{Env.DB_SCHEMA}\".{platform_key_table}
+        WHERE key = '{token}'
+    """
```

Ruff (0.14.13)

Possible SQL injection vector through string-based query construction

(S608)